src/Optional/Plots.jl

```diff
@@ -0,0 +1,33 @@
+using .Plots
+
+preprocess(name, val::Plots.Plot, data) = push!(data, name=>val)
+summary_impl(name, value::Plots.Plot) = image_summary(name, value)
```

Comment:
I guess this can't be in https://github.com/PhilipVinc/TensorBoardLogger.jl/blob/master/src/logger_dispatch.jl
since it depends on plots.

Unless we wanted to move the require block there.

After this is merged i might have a play with a futher tweaked structure of files and see how that looks

Reply:
If you have better proposals, I'm all ears.

Reply:
I think this is fine, but I will let you now if I find something nicer